new/prusti-interface/src/specs/typed.rs

```diff
@@ -42,6 +44,24 @@ impl StructuralToTyped<Expression> for json::Expression {
     }
 }
 
+impl StructuralToTyped<TriggerSet> for json::TriggerSet {
+    fn to_typed(self, typed_expressions: &HashMap<String, rustc_hir::BodyId>) -> TriggerSet {
+        TriggerSet {
+            0: vec![]
```

Comment:
`TriggerSet(Vec::new())`

Reply:
does not work: `expected function, tuple struct or tuple variant, found type alias `TriggerSet``

Reply:
How about `Self(Vec::new())`?

Reply:
expected struct `prusti_specs::specifications::common::TriggerSet`, found struct `prusti_specs::specifications::json::TriggerSet`

Reply:
Ok.